site/topics/memory/esap-ram.rst

```diff
@@ -0,0 +1,32 @@
+************************
+Address Register and RAM
+************************
+
+* RAM stores data and instructions for the computer
```

Comment:
Maybe important to distinguish that memory is ephemeral, not persistent. It's relevant to the circuit design imo.